packages/react/src/components/builder-component.component.tsx

```diff
@@ -416,7 +418,9 @@ export class BuilderComponent extends React.Component<
     if (Builder.isBrowser) {
       const key = this.props.apiKey;
       if (key && key !== this.builder.apiKey) {
-        this.builder.apiKey = key;
+        // hmm this could cause issues with global symbols
```

Comment:
Can you add more context to this comment or remove it? As it stands now it is kind of cryptic 😄